lib/assets/sass/prototype.scss

```diff
@@ -1,3 +1,5 @@
+$govuk-new-typography-scale: true;
```

Comment:
Happy to see new typography scale turned on for the 'manage prototype' area but I don't think we should be enabling it in the prototypes themselves.